package.json

```diff
@@ -182,7 +185,9 @@
     ],
     "moduleNameMapper": {
       "\\.(css|scss)$": "identity-obj-proxy",
-      "^(.*)\\?worker&inline$": "$1"
+      "^(.*)\\?worker&inline$": "$1",
+      "^react-markdown$": "<rootDir>/src/components/html-renderer/__mocks__/react-markdown.js",
```

Comment:
When Jest tried to run the tests, it couldn't understand the ES module syntax, especially from react-markdown. So the work around is to use these mocks, which seems to provide enough functionality to test component's props, rendering, and basic functionality